src/fastoad/openmdao/exceptions.py

```diff
@@ -16,22 +14,37 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+from os import PathLike
+from typing import Iterable
 
-from typing import List
-
+from fastoad._utils.files import as_path
 from fastoad.exceptions import FastError
 
 
-class FASTOpenMDAONanInInputFile(FastError):
+#  This file is part of FAST-OAD : A framework for rapid Overall Aircraft Design
+#  Copyright (C) 2024 ONERA & ISAE-SUPAERO
+#  FAST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+class FASTOpenMDAONanInInputsError(FastError):
```

Comment:
I know it was already there in the previous exception name but why does it start with FASTOpenMDAO ? Is it FAST or OpenMDAO. Also being very picky about things that don't matter, since NAN is already an acronym (like MDAO) why isn't it all uppercase ?

Reply:
Indeed, I didn't pay attention to this part of the name, but it does not make much sense here. Fixed in 72651bf030753de7f83546b2324f70a522186ad1
And for the "picky" part ;), the standard acronym is "NaN", but I feel that using it would make the CamelCase messy.